From 45dd1ac82881153c73bd4243cd20f3b13955ad21 Mon Sep 17 00:00:00 2001
From: Chee Hong Ang <chee.hong.ang@intel.com>
Date: Sat, 11 May 2019 00:09:46 +0800
Subject: [PATCH] ARM: socfpga: Stratix10: Disable CONFIG_PSCI_RESET

Avoid invoking 'SYSTEM_RESET' PSCI function because PSCI
function calls are not supported in u-boot running in EL3.

Signed-off-by: Chee Hong Ang <chee.hong.ang@intel.com>
---
 arch/arm/cpu/armv8/Kconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/cpu/armv8/Kconfig b/arch/arm/cpu/armv8/Kconfig
index 7405c3a4a1..409ee7ada0 100644
--- a/arch/arm/cpu/armv8/Kconfig
+++ b/arch/arm/cpu/armv8/Kconfig
@@ -108,7 +108,8 @@ config PSCI_RESET
 		   !TARGET_LS1046ARDB && !TARGET_LS1046AQDS && \
 		   !TARGET_LS2081ARDB && !TARGET_LX2160ARDB && \
 		   !TARGET_LX2160AQDS && \
-		   !ARCH_UNIPHIER && !TARGET_S32V234EVB
+		   !ARCH_UNIPHIER && !TARGET_S32V234EVB && \
+		   !TARGET_SOCFPGA_STRATIX10
 	help
 	  Most armv8 systems have PSCI support enabled in EL3, either through
 	  ARM Trusted Firmware or other firmware.
-- 
2.13.0

